.github/workflows/workflow-executor.yaml

```diff
@@ -56,7 +56,7 @@ on:
       environment:
         description: |
           Any required environment variables to set for the speakeasy run execution. Set as a string with each entry of the form "key=value".
-  
+
```

Comment:
intentional?

Reply:
Guessing that was my editor automatic formatting. Doesn't seem like the whitespace there would have any effect